config/default.yml

```diff
@@ -1,3 +1,7 @@
+Salsify/DandelionApplicationSerializer:
```

Comment:
Should we call this cop `ApplicationSerializer` in case we introduce that convention in other projects?